src/shogun/converter/ica/FastICA.cpp

```diff
@@ -130,7 +130,11 @@ CFeatures* CFastICA::apply(CFeatures* features)
 
 	float64_t lim = tol+1;
 	for (auto i : progress(
-	         range(0, max_iter), "PROGRESS: ", UTF8, [&] { return lim > tol; }))
+	    range(0, max_iter),
+		"PROGRESS: ",
```

Comment:
Put this line and the two below at the same indentation than range, like in the doc we were referring to.